oaknational/Oak-Web-Application
# LESQ-264 Feat/home page tabs

## Description

- List of changes

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-1909--oak-web-application.netlify.thenational.academy/teachers
2. Click on _______
3. You should see _______

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
